config/201-clusterrole.yaml

```diff
@@ -0,0 +1,111 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: eventing-sources-controller
+rules:
+# Sources admin
+- apiGroups:
+  - sources.eventing.knative.dev
+  resources:
+  - awssqssources
+  - containersources
+  - cronjobsources
+  - githubsources
+  - kuberneteseventsources
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+  
+# Source statuses update
+- apiGroups:
+  - sources.eventing.knative.dev
+  resources:
+  - awssqssources/status
```

Comment:
get/patch/update are the only verbs supported by foo/status. do you have an example of them mixed?

Reply:
I don't, but I'm trying to prevent it from happening in the future. The migration to /status in knative/eventing caused quite a few to be missed. Is it problematic to give the additional permissions on the /status object?

Reply:
I don't think you want to attempt to assign delete to /status